Formula/doggo.rb

```diff
@@ -2,18 +2,11 @@ class Doggo < Formula
   desc "Command-line DNS Client for Humans"
   homepage "https://doggo.mrkaran.dev/"
   license "GPL-3.0-or-later"
-  revision 1
   head "https://github.com/mr-karan/doggo.git", branch: "main"
 
   stable do
-    url "https://github.com/mr-karan/doggo/archive/refs/tags/v0.5.5.tar.gz"
-    sha256 "7ba1340ce46566ca8fa1565ef261519dee5b1c7007aea97eb1f9329f8a3f0403"
-
-    # patch for quic-go, remove when it is available
-    patch do
-      url "https://github.com/mr-karan/doggo/commit/7db5c2144fa4a3f18afe1c724b9367b03f84aed7.patch?full_index=1"
-      sha256 "e34f71e6618817edfd0d7b01657a407e9573e74a7ea89d048abc95bac46bb21e"
-    end
+    url "https://github.com/mr-karan/doggo/archive/refs/tags/v0.5.6.tar.gz"
+    sha256 "1965f4c909991bc38b65784ccbc03f4760214bca34f1bb984999f1fc8714ff96"
```

Comment:
Since we no longer need the patches you can move these two lines to just below the `homepage` line above. You can then delete the `stable` block (since it will be empty).

Reply:
Tnx, done in f0f5395d648c00ca6539a196f3367866e5df1a72